mLRS/Common/common_conf.h

```diff
@@ -103,6 +103,10 @@
 #define MODE_31HZ_SEND_FRAME_TMO        15 // just needs to be larger than toa, not critical
 #define MODE_19HZ_SEND_FRAME_TMO        25 // just needs to be larger than toa, not critical
 
+#define FHSS_NUM_BAND_433_MHZ               3 // there are only 3
```

Comment:
pl swap with _19HZ_MODE, so we have equal sequence everywhere

ALSO: can be only 2 (one is needed for bind, and by current design bind cannot be hopped on)(I consider using one of the "bad" frequencies for bind, so that we can have 3 for hopping, but that's futur LOL)

Reply:
Order swapped, updated 2 channels only.